examples/jss/src/index.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { render } from 'react-dom';
 
-import './global-styles';
+import GlobalStyles from './components/global-styles';
 import App from './components/app';
 
-render(<App />, document.getElementById('example-root'));
+render(<GlobalStyles><App /></GlobalStyles>, document.getElementById('example-root'));
```

Comment:
Not everything in JSX corresponds 1:1 the custom-elements. It may be not as obvious from rendering perspective, but I think it is typical in react eco system.